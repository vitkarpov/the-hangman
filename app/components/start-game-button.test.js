const StartGameButton = require("./start-game-button");

describe("StartGameButton", () => {
  it("renders correctly", () => {
    expect(new StartGameButton().render()).toMatchSnapshot();
  });
});
