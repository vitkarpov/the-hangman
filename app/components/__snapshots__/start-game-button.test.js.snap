// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StartGameButton renders correctly 1`] = `"<button class=\\"start-button\\">Go</button>"`;
